### PR TITLE
prepare elrond and seednode dockerfiles

### PR DESCRIPTION
**Pull request by BenWhiteJam**
- include Docker builder pattern (multi-stage builds)
- golang base container
- start with Entrypoint
Elrond Node:
- remove network specific configs
TODO: remove version dependencies for arwen-wasm-vm

### CODE REVIEW

File: docker/elrond/Dockerfile [hunk 1]
```diff
@@ -1,29 +1,18 @@
...
-RUN ls
-
-COPY ./ ./elrond-go/
-WORKDIR elrond-go
-RUN >> skipP2PMessengerTests
-RUN ls
-RUN GO111MODULE=on go mod vendor
\ No newline at end of file
+# ===== SECOND STAGE ======
+FROM ubuntu
```
Comment (valentin-lup) [inline docker/elrond/Dockerfile:12]: Should we maybe lock ubuntu to 18.04 just to be on par with the other testnets ?

Comment (ccorcoveanu) [reply docker/elrond/Dockerfile:12]: Yup, this is my only remark too 👍

Comment (BenWhiteJam) [reply docker/elrond/Dockerfile:12]: sure, totally makes sense

**Comment by BenWhiteJam** [comment]
closing this one in regards to #2052

**Pull request closed without merging**